device.go

```diff
@@ -152,3 +152,16 @@ func (d *device) CommitUpdate() error {
 	// For now set only appropriate boot flags
 	return d.WriteEnv(BootVars{"upgrade_available": "0"})
 }
+
+func (d *device) HasUpdate() (bool, error) {
```

Comment:
Am I right that this goes together with the commit "mender: all BootEnv handling moved to device"? Will that commit even compile without this? In general I think such functionality moves should either be grouped in the same commit or ordered in a way so that the code always compiles.

(Github doesn't always order things correctly though, so apologies if I'm wrong)